isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend/serviceschicht.adoc

```diff
@@ -0,0 +1,119 @@
+= Serviceschicht
+
+include::documentation-guide::partial$licence.adoc[]
+
+// tag::serviceschicht-definition[]
+
+Die Serviceschicht stellt Services von Backends innerhalb einer Systemlandschaft zur Verfügung.
+Sie regelt die Autorisierung von Aufrufen, validiert die übergebenen Daten und ruft die entsprechenden fachlichen Operationen des Anwendungskerns auf.
+Der Aufbau der Serviceschicht sowie ihre Kommunikation mit anderen Schichten ist in <<systemarchitektur-serviceschicht,folgender Grafik>> dargestellt.
+
+[[systemarchitektur-serviceschicht]]
+.Einbettung der Serviceschicht in die Systemarchitektur
+image::software-technisch/backend/service.dn.svg[width=80%,align=center]
+
+// end::serviceschicht-definition[]
+
+[[fachkomponenten]]
+== Fachkomponenten
+
+// tag::fachkomponente-definition[]
+
+Die *Fachkomponenten* der Serviceschicht spiegeln die Fachkomponenten aus dem Anwendungskern wider.
+Sie nehmen die Daten der Anfragen von außen entgegen und stellen die Funktionalität und Daten des Anwendungskerns nach außen hin zur Verfügung.
+Zu den Aufgaben der Fachkomponenten gehören maßgeblich:
```

Comment:
Diesen Satz und die unten stehende Aufzählung der Aufgaben sollte man vielleicht in den Abschnitt [[fachkomponente-aufgaben]] verschieben

Reply:
Hier geht es um eine knappe Definition der Fachkomponente der Serviceschicht, die an anderen Stellen der Dokumentation eingebunden werden kann. Später folgt die ausführliche Beschreibung. Daher ist ein gewisse Dopplung vorhanden.

Reply:
Die Dopplung ist OK. Im Text zur Überschrift des Abschnitts Fachkomponenten sollte jedoch nichts stehen was ausschließlich in genau zu einem der Unterabschnitte gehört